posthog/api/organization_feature_flag.py

```diff
@@ -96,10 +100,64 @@ def copy_flags(self, request, *args, **kwargs):
                 )
                 continue
 
-            context = {
-                "request": request,
-                "team_id": target_project_id,
-            }
+            # get all linked cohorts, sorted by creation order
+            seen_cohorts_cache: Dict[int, Cohort] = {}
+            sorted_cohort_ids = flag_to_copy.get_cohort_ids(
+                seen_cohorts_cache=seen_cohorts_cache, sort_by_topological_order=True
+            )
+
+            # destination cohort id is different from original cohort id - create mapping
+            name_to_dest_cohort_id: Dict[str, int] = {}
+            # create cohorts in the destination project
+            if len(sorted_cohort_ids):
+                for cohort_id in sorted_cohort_ids:
+                    original_cohort = seen_cohorts_cache[cohort_id]
+
+                    # search in destination project by name
+                    destination_cohort = Cohort.objects.filter(
+                        name=original_cohort.name, team_id=target_project_id, deleted=False
+                    ).first()
+                    # create new cohort in the destination project
+                    if not destination_cohort:
+                        prop_group = Filter(
```

Comment:
You can replace this with `original_cohort.properties`

Reply:
I think we can't - _original_cohort.properties_ references the original cohort in _seen_cohorts_cache_. Overwriting it raises this error down the line: `posthog.models.cohort.cohort.Cohort.DoesNotExist: Cohort matching query does not exist`

(I don't quite understand this)

Reply:
ooh, very good point indeed!

You can create a new PropertyGroup using `original_cohort.properties` : https://github.com/PostHog/posthog/blob/master/posthog/models/property/property.py#L304-L310

We need to use `original_cohort.properties` because that handles the backwards compatibility case with `cohort.groups` while the current code doesn't.

Reply:
...  although, that might be passing in just the original list, annoying, so maybe serialise and deserialise - use `original_cohort.properties.to_dict()` and then convert that back into a property group using the filter

Reply:
cool thanks!